web/src/app/schedules/ScheduleTZFilter.js

```diff
@@ -21,14 +22,15 @@ export function ScheduleTZFilter(props) {
     variables: { id: props.scheduleID },
   })
 
-  let label, tz
+  let label, tz, short
   if (error) {
     label = 'Error: ' + (error.message || error)
   } else if (!data && loading) {
     label = 'Fetching timezone information...'
   } else {
     tz = data.schedule.timeZone
-    label = props.label ? props.label(tz) : `Show times in ${tz}`
+    short = DateTime.fromObject({ zone: tz }).toFormat('ZZZZ')
+    label = props.label ? props.label(tz) : `Show times in ${tz} (${short})`
```

Comment:
We should pass the short bit into `props.label` as well so we display consistently.

```suggestion
    let tzName = tz
    if (short !== tz) tz += ` (${short})`
    label = props.label ? props.label(tzName) : `Show times in ${tzName}`
```